runelite-api/src/main/java/net/runelite/api/MenuEntry.java

```diff
@@ -36,10 +36,33 @@
 	private int param0;
 	private int param1;
 
+	public MenuEntry(String option, String target, int identifier, int type, int param0, int param1)
```

Comment:
You should instead create builder

Reply:
I am not sure what you mean by this.

This constructor is not used right now, I just thought it should be since we seem to be needing more ways to create MenuEntry items.